microcosm_flask/conventions/landing.py

```diff
@@ -83,18 +69,18 @@ def render_landing_page():
 
         """
         properties = get_properties_and_version()
-        description = get_description(properties)
         swagger_versions = get_swagger_versions()
         config = graph.config_convention.to_dict()
         health = graph.health_convention.to_dict()
         env = get_env_file_commands(config, graph.metadata.name)
 
         return Template(template).render(
             config=pretty_dict(config),
-            description=description,
+            description=properties.description,
             env=env,
             health=pretty_dict(health),
             homepage=getattr(properties, 'url', None),
+            links=graph.config.landing_convention.get("links", []),
```

Comment:
We're starting to pull input data from a wide variety of places without any obvious consistency. Do you think we're ready for an abstraction layer (e.g. a controller) and object model between this route and the template?

Reply:
That would enable us to retrieve these links from an endpoint instead of getting it directly from config?

Doing that would enable us to have logic for generating the url? Ex. a link to a grafana dashboard

Making the coupling really loose makes sense to me now but that means that all url logic will need to be in ansible.

I don't know the answer.